src/pages/tabs/tabs.ts

```diff
@@ -11,8 +10,7 @@ export class TabsPage {
   // this tells the tabs component which Pages
   // should be each tab's root Page
   tab1Root: any = HomePage;
-  tab2Root: any = AboutPage;
-  tab3Root: any = ContactPage;
+  tab2Root: any = InventoryPage;
```

Comment:
Would it make sense to give both tab roots more descriptive names, like `inventoryTabRoot`?